packages/laconia-adapter-api/src/websocket.js

```diff
@@ -0,0 +1,12 @@
+const { res, parseWebsocket } = require("@laconia/event").apigateway;
+
+const websocketAdapter = app => async (...args) => {
```

Comment:
- Can we keep the convention consistent by renaming this file to createWebSocketAdapter?
- Should args be renamed to just `event`?
- I think there is missing a unit test too for this one

Reply:
As far as I understood it, the app function can also get a context, not only an event, hence passing through alls args.

Reply:
I forgot to spread the args of course.